docs/tutorials/query-cache.txt

```diff
@@ -215,20 +215,11 @@ System Collections
 ==================
 
 MongoDB stores system information in collections that use the ``database.system.*``
-namespace pattern.
+namespace pattern. These are called system collections.
 
-When the query cache is enabled, it will cache all system collection results.
-However, system collections are not subject to the same cache invalidation rules
-as normal collections; if your app relies on up-to-date information from
-system collections, please ensure that you are performing queries against the
-system collections with the query cache disabled, as demonstrated below.
+Query caching has been completely disabled on system collections in order to
+prevent the cache from returning stale data.
```

Comment:
Do you think more explanation is necessary here? I didn't think it would be helpful to get into the details of it all for user documentation, but I would be open to including more detail.

Reply:
How about the following instead:

Data in system collections can change due to activity not triggered by the application (such as internal server processes) and as a result of a variety of database commands issued by the application. Because of the difficulty of determining when the cached results for system collections should be expired, queries on system collections bypass the query cache.

Reply:
Perfect, thanks for the suggestion.